cmd/bootstrap/main.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2016 Mirantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bufio"
+	"encoding/json"
+	"log"
+	"os"
+
+	"k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+
+	"github.com/Mirantis/k8s-AppController/client"
+)
+
+func getFileContents(stream *os.File) string {
+	result := ""
+	scanner := bufio.NewScanner(stream)
+	for scanner.Scan() {
+		result += scanner.Text() + "\n"
+	}
+	return result
+}
+
+func createTPRIfNotExists(tpr *extensions.ThirdPartyResource, client *unversioned.Client) {
```

Comment:
Can we get rid of the pointer here. I know we might use pointers for the sake of efficiency, but we only call this twice in the pod's lifecycle. And I'd personally like to use pointers when it is strictly necessarily, i.e. when we change stuff, or when we move big chunks  of memory too much.